apps/explorer/src/pages/epochs/EpochDetail.tsx

```diff
@@ -5,7 +5,7 @@ import { useFeature, useGrowthBook } from '@growthbook/growthbook-react';
 import { Navigate } from 'react-router-dom';
 
 import { validatorsTableData } from '../validators/Validators';
-import { CheckpointsTable } from './CheckpointsTable';
+// import { CheckpointsTable } from '../checkpoints/CheckpointsTable';
```

Comment:
is this intentional?

Reply:
more or less :) it was previously just a table of mock data - until we can see the start / end sequence numbers in an epoch (coming soon) we can't implement the checkpoints table